app/controllers/company/WhatTypeBusinessController.scala

```diff
@@ -14,16 +14,18 @@
  * limitations under the License.
  */
 
-package controllers
```

Comment:
This page should sit outside the company package because it is common between the company and partnership